{"uuid":"faa93c4b-1dd4-409a-bbee-d1abe8eda3f8","name":"Default suite","start":1521415029598,"stop":1521415052117,"children":["126d076a-b370-418e-b4f1-38b216d96f4c"],"befores":[{"name":"beforeSuite","status":"passed","stage":"finished","description":"","start":1521415029613,"stop":1521415029692},{"name":"setUp","status":"broken","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"System.ArgumentOutOfRangeException: Appium request /session resulted in an error: An unknown server-side error occurred while processing the command. Original error: Cannot read manifest information from '/usr/local/deviceconnect/Appium/dc-appium-ebaac7a8-1.7.1/node_modules/appium/node_modules/appium-uiautomator2-driver/uiautomator2/appium-uiautomator2-server-v0.1.8.apk'. Does the package exist and is accessible?\nParameter name: status\n13\n  at MobileLabs.AirStream.Framework.Async.IO.AppiumClient.ExceptionIfNonZeroStatus (System.String url, System.String response) [0x00000] in <filename unknown>:0 \n  at MobileLabs.AirStream.Framework.Async.IO.AppiumClient+<RequestUninitialized>d__27.MoveNext () [0x00000] in <filename unknown>:0 \n--- End of stack trace from previous location where exception was thrown ---\n  at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw () [0x0000b] in /Users/eiz/code/mono/mcs/class/corlib/System.Runtime.ExceptionServices/ExceptionDispatchInfo.cs:62 \n  at System.Runtime.CompilerServices.TaskAwaiter`1[System.String].GetResult () [0x00034] in /Users/eiz/code/mono/mcs/class/corlib/System.Runtime.CompilerServices/TaskAwaiter_T.cs:59 \n  at MobileLabs.AirStream.Framework.Async.IO.AppiumClient+<RequestUninitialized>d__29`1[MobileLabs.AirStream.Framework.Async.IO.AppiumResponse`1[System.Collections.Generic.Dictionary`2[System.String,System.Object]]].MoveNext () [0x00000] in <filename unknown>:0 \n--- End of stack trace from previous location where exception was thrown ---\n  at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw () [0x0000b] in /Users/eiz/code/mono/mcs/class/corlib/System.Runtime.ExceptionServices/ExceptionDispatchInfo.cs:62 \n  at System.Runtime.CompilerServices.TaskAwaiter`1[MobileLabs.AirStream.Framework.Async.IO.AppiumResponse`1[System.Collections.Generic.Dictionary`2[System.String,System.Object]]].GetResult () [0x00034] in /Users/eiz/code/mono/mcs/class/corlib/System.Runtime.CompilerServices/TaskAwaiter_T.cs:59 \n  at MobileLabs.AirStream.Framework.Async.IO.AppiumClient+<InitializeAsync>d__21.MoveNext () [0x00000] in <filename unknown>:0 \n--- End of stack trace from previous location where exception was thrown ---\n  at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw () [0x0000b] in /Users/eiz/code/mono/mcs/class/corlib/System.Runtime.ExceptionServices/ExceptionDispatchInfo.cs:62 \n  at System.Runtime.CompilerServices.TaskAwaiter.GetResult () [0x00034] in /Users/eiz/code/mono/mcs/class/corlib/System.Runtime.CompilerServices/TaskAwaiter.cs:60 \n  at MobileLabs.AirStream.Framework.Async.IO.Appium.AppiumSession+<StartAppiumSession>d__107.MoveNext () [0x00000] in <filename unknown>:0 \n--- End of stack trace from previous location where exception was thrown ---\n  at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw () [0x0000b] in /Users/eiz/code/mono/mcs/class/corlib/System.Runtime.ExceptionServices/ExceptionDispatchInfo.cs:62 \n  at System.Runtime.CompilerServices.TaskAwaiter`1[MobileLabs.AirStream.Gateway.Protocol.AppiumStartServiceResult].GetResult () [0x00034] in /Users/eiz/code/mono/mcs/class/corlib/System.Runtime.CompilerServices/TaskAwaiter_T.cs:59 \n  at MobileLabs.AirStream.Framework.Async.IO.Appium.AppiumSession+<Initialize>d__92.MoveNext () [0x00000] in <filename unknown>:0 \n--- End of stack trace from previous location where exception was thrown ---\n  at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw () [0x0000b] in /Users/eiz/code/mono/mcs/class/corlib/System.Runtime.ExceptionServices/ExceptionDispatchInfo.cs:62 \n  at System.Runtime.CompilerServices.TaskAwaiter.GetResult () [0x00034] in /Users/eiz/code/mono/mcs/class/corlib/System.Runtime.CompilerServices/TaskAwaiter.cs:60 \n  at MobileLabs.AirStream.Framework.Async.IO.AppiumClient+<Request>d__24`1[MobileLabs.AirStream.Framework.Async.IO.AppiumResponse`1[System.Object]].MoveNext () [0x00000] in <filename unknown>:0 \n--- End of stack trace from previous location where exception was thrown ---\n  at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw () [0x0000b] in /Users/eiz/code/mono/mcs/class/corlib/System.Runtime.ExceptionServices/ExceptionDispatchInfo.cs:62 \n  at MobileLabs.AirStream.Framework.Async.IO.Appium.AppiumSession+<Initialize>d__92.MoveNext () [0x00000] in <filename unknown>:0 \n--- End of stack trace from previous location where exception was thrown ---\n  at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw () [0x0000b] in /Users/eiz/code/mono/mcs/class/corlib/System.Runtime.ExceptionServices/ExceptionDispatchInfo.cs:62 \n  at System.Runtime.CompilerServices.TaskAwaiter.GetResult () [0x00034] in /Users/eiz/code/mono/mcs/class/corlib/System.Runtime.CompilerServices/TaskAwaiter.cs:60 \n  at MobileLabs.AirStream.Framework.Async.IO.Appium.AppiumSession+<Create>d__90.MoveNext () [0x00000] in <filename unknown>:0 \n--- End of stack trace from previous location where exception was thrown ---\n  at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw () [0x0000b] in /Users/eiz/code/mono/mcs/class/corlib/System.Runtime.ExceptionServices/ExceptionDispatchInfo.cs:62 \n  at System.Runtime.CompilerServices.TaskAwaiter`1[MobileLabs.AirStream.Framework.Async.IO.Appium.AppiumSession].GetResult () [0x00034] in /Users/eiz/code/mono/mcs/class/corlib/System.Runtime.CompilerServices/TaskAwaiter_T.cs:59 \n  at MobileLabs.AirStream.Web.Controllers.AppiumController+<CreateSession>d__13.MoveNext () [0x00000] in <filename unknown>:0 \n--- End of stack trace from previous location where exception was thrown ---\n  at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw () [0x0000b] in /Users/eiz/code/mono/mcs/class/corlib/System.Runtime.ExceptionServices/ExceptionDispatchInfo.cs:62 \n  at System.Runtime.CompilerServices.TaskAwaiter`1[MobileLabs.AirStream.Framework.IO.Mvc.MvcJsonResult].GetResult () [0x00034] in /Users/eiz/code/mono/mcs/class/corlib/System.Runtime.CompilerServices/TaskAwaiter_T.cs:59 \n  at MobileLabs.AirStream.Web.Controllers.AppiumController+<HandleRequestAsync>d__10.MoveNext () [0x00000] in <filename unknown>:0  (WARNING: The server did not provide any stacktrace information)\nCommand duration or timeout: 21.34 seconds\nBuild info: version: '3.7.1', revision: '8a0099a', time: '2017-11-06T21:01:39.354Z'\nSystem info: host: 'LD-4V2P3G2', ip: '192.168.121.171', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_161'\nDriver info: driver.version: RFWebsiteDriver","trace":"org.openqa.selenium.WebDriverException: System.ArgumentOutOfRangeException: Appium request /session resulted in an error: An unknown server-side error occurred while processing the command. Original error: Cannot read manifest information from '/usr/local/deviceconnect/Appium/dc-appium-ebaac7a8-1.7.1/node_modules/appium/node_modules/appium-uiautomator2-driver/uiautomator2/appium-uiautomator2-server-v0.1.8.apk'. Does the package exist and is accessible?\nParameter name: status\n13\n  at MobileLabs.AirStream.Framework.Async.IO.AppiumClient.ExceptionIfNonZeroStatus (System.String url, System.String response) [0x00000] in <filename unknown>:0 \n  at MobileLabs.AirStream.Framework.Async.IO.AppiumClient+<RequestUninitialized>d__27.MoveNext () [0x00000] in <filename unknown>:0 \n--- End of stack trace from previous location where exception was thrown ---\n  at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw () [0x0000b] in /Users/eiz/code/mono/mcs/class/corlib/System.Runtime.ExceptionServices/ExceptionDispatchInfo.cs:62 \n  at System.Runtime.CompilerServices.TaskAwaiter`1[System.String].GetResult () [0x00034] in /Users/eiz/code/mono/mcs/class/corlib/System.Runtime.CompilerServices/TaskAwaiter_T.cs:59 \n  at MobileLabs.AirStream.Framework.Async.IO.AppiumClient+<RequestUninitialized>d__29`1[MobileLabs.AirStream.Framework.Async.IO.AppiumResponse`1[System.Collections.Generic.Dictionary`2[System.String,System.Object]]].MoveNext () [0x00000] in <filename unknown>:0 \n--- End of stack trace from previous location where exception was thrown ---\n  at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw () [0x0000b] in /Users/eiz/code/mono/mcs/class/corlib/System.Runtime.ExceptionServices/ExceptionDispatchInfo.cs:62 \n  at System.Runtime.CompilerServices.TaskAwaiter`1[MobileLabs.AirStream.Framework.Async.IO.AppiumResponse`1[System.Collections.Generic.Dictionary`2[System.String,System.Object]]].GetResult () [0x00034] in /Users/eiz/code/mono/mcs/class/corlib/System.Runtime.CompilerServices/TaskAwaiter_T.cs:59 \n  at MobileLabs.AirStream.Framework.Async.IO.AppiumClient+<InitializeAsync>d__21.MoveNext () [0x00000] in <filename unknown>:0 \n--- End of stack trace from previous location where exception was thrown ---\n  at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw () [0x0000b] in /Users/eiz/code/mono/mcs/class/corlib/System.Runtime.ExceptionServices/ExceptionDispatchInfo.cs:62 \n  at System.Runtime.CompilerServices.TaskAwaiter.GetResult () [0x00034] in /Users/eiz/code/mono/mcs/class/corlib/System.Runtime.CompilerServices/TaskAwaiter.cs:60 \n  at MobileLabs.AirStream.Framework.Async.IO.Appium.AppiumSession+<StartAppiumSession>d__107.MoveNext () [0x00000] in <filename unknown>:0 \n--- End of stack trace from previous location where exception was thrown ---\n  at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw () [0x0000b] in /Users/eiz/code/mono/mcs/class/corlib/System.Runtime.ExceptionServices/ExceptionDispatchInfo.cs:62 \n  at System.Runtime.CompilerServices.TaskAwaiter`1[MobileLabs.AirStream.Gateway.Protocol.AppiumStartServiceResult].GetResult () [0x00034] in /Users/eiz/code/mono/mcs/class/corlib/System.Runtime.CompilerServices/TaskAwaiter_T.cs:59 \n  at MobileLabs.AirStream.Framework.Async.IO.Appium.AppiumSession+<Initialize>d__92.MoveNext () [0x00000] in <filename unknown>:0 \n--- End of stack trace from previous location where exception was thrown ---\n  at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw () [0x0000b] in /Users/eiz/code/mono/mcs/class/corlib/System.Runtime.ExceptionServices/ExceptionDispatchInfo.cs:62 \n  at System.Runtime.CompilerServices.TaskAwaiter.GetResult () [0x00034] in /Users/eiz/code/mono/mcs/class/corlib/System.Runtime.CompilerServices/TaskAwaiter.cs:60 \n  at MobileLabs.AirStream.Framework.Async.IO.AppiumClient+<Request>d__24`1[MobileLabs.AirStream.Framework.Async.IO.AppiumResponse`1[System.Object]].MoveNext () [0x00000] in <filename unknown>:0 \n--- End of stack trace from previous location where exception was thrown ---\n  at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw () [0x0000b] in /Users/eiz/code/mono/mcs/class/corlib/System.Runtime.ExceptionServices/ExceptionDispatchInfo.cs:62 \n  at MobileLabs.AirStream.Framework.Async.IO.Appium.AppiumSession+<Initialize>d__92.MoveNext () [0x00000] in <filename unknown>:0 \n--- End of stack trace from previous location where exception was thrown ---\n  at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw () [0x0000b] in /Users/eiz/code/mono/mcs/class/corlib/System.Runtime.ExceptionServices/ExceptionDispatchInfo.cs:62 \n  at System.Runtime.CompilerServices.TaskAwaiter.GetResult () [0x00034] in /Users/eiz/code/mono/mcs/class/corlib/System.Runtime.CompilerServices/TaskAwaiter.cs:60 \n  at MobileLabs.AirStream.Framework.Async.IO.Appium.AppiumSession+<Create>d__90.MoveNext () [0x00000] in <filename unknown>:0 \n--- End of stack trace from previous location where exception was thrown ---\n  at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw () [0x0000b] in /Users/eiz/code/mono/mcs/class/corlib/System.Runtime.ExceptionServices/ExceptionDispatchInfo.cs:62 \n  at System.Runtime.CompilerServices.TaskAwaiter`1[MobileLabs.AirStream.Framework.Async.IO.Appium.AppiumSession].GetResult () [0x00034] in /Users/eiz/code/mono/mcs/class/corlib/System.Runtime.CompilerServices/TaskAwaiter_T.cs:59 \n  at MobileLabs.AirStream.Web.Controllers.AppiumController+<CreateSession>d__13.MoveNext () [0x00000] in <filename unknown>:0 \n--- End of stack trace from previous location where exception was thrown ---\n  at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw () [0x0000b] in /Users/eiz/code/mono/mcs/class/corlib/System.Runtime.ExceptionServices/ExceptionDispatchInfo.cs:62 \n  at System.Runtime.CompilerServices.TaskAwaiter`1[MobileLabs.AirStream.Framework.IO.Mvc.MvcJsonResult].GetResult () [0x00034] in /Users/eiz/code/mono/mcs/class/corlib/System.Runtime.CompilerServices/TaskAwaiter_T.cs:59 \n  at MobileLabs.AirStream.Web.Controllers.AppiumController+<HandleRequestAsync>d__10.MoveNext () [0x00000] in <filename unknown>:0  (WARNING: The server did not provide any stacktrace information)\nCommand duration or timeout: 21.34 seconds\nBuild info: version: '3.7.1', revision: '8a0099a', time: '2017-11-06T21:01:39.354Z'\nSystem info: host: 'LD-4V2P3G2', ip: '192.168.121.171', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_161'\nDriver info: driver.version: RFWebsiteDriver\r\n\tat sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)\r\n\tat sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)\r\n\tat sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)\r\n\tat java.lang.reflect.Constructor.newInstance(Unknown Source)\r\n\tat org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:214)\r\n\tat org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:166)\r\n\tat org.openqa.selenium.remote.JsonWireProtocolResponse.lambda$new$0(JsonWireProtocolResponse.java:53)\r\n\tat org.openqa.selenium.remote.JsonWireProtocolResponse.lambda$getResponseFunction$2(JsonWireProtocolResponse.java:91)\r\n\tat org.openqa.selenium.remote.ProtocolHandshake.lambda$createSession$0(ProtocolHandshake.java:123)\r\n\tat java.util.stream.ReferencePipeline$3$1.accept(Unknown Source)\r\n\tat java.util.Spliterators$ArraySpliterator.tryAdvance(Unknown Source)\r\n\tat java.util.stream.ReferencePipeline.forEachWithCancel(Unknown Source)\r\n\tat java.util.stream.AbstractPipeline.copyIntoWithCancel(Unknown Source)\r\n\tat java.util.stream.AbstractPipeline.copyInto(Unknown Source)\r\n\tat java.util.stream.AbstractPipeline.wrapAndCopyInto(Unknown Source)\r\n\tat java.util.stream.FindOps$FindOp.evaluateSequential(Unknown Source)\r\n\tat java.util.stream.AbstractPipeline.evaluate(Unknown Source)\r\n\tat java.util.stream.ReferencePipeline.findFirst(Unknown Source)\r\n\tat org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:126)\r\n\tat org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:73)\r\n\tat org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:142)\r\n\tat io.appium.java_client.remote.AppiumCommandExecutor.execute(AppiumCommandExecutor.java:89)\r\n\tat org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:600)\r\n\tat io.appium.java_client.DefaultGenericMobileDriver.execute(DefaultGenericMobileDriver.java:42)\r\n\tat io.appium.java_client.AppiumDriver.execute(AppiumDriver.java:1)\r\n\tat io.appium.java_client.android.AndroidDriver.execute(AndroidDriver.java:1)\r\n\tat org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:219)\r\n\tat org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:142)\r\n\tat io.appium.java_client.DefaultGenericMobileDriver.<init>(DefaultGenericMobileDriver.java:38)\r\n\tat io.appium.java_client.AppiumDriver.<init>(AppiumDriver.java:83)\r\n\tat io.appium.java_client.AppiumDriver.<init>(AppiumDriver.java:93)\r\n\tat io.appium.java_client.android.AndroidDriver.<init>(AndroidDriver.java:72)\r\n\tat com.rf.core.driver.website.RFWebsiteDriver.loadApplication(RFWebsiteDriver.java:107)\r\n\tat com.rf.test.website.RFLSDWebsiteBaseTest.setUp(RFLSDWebsiteBaseTest.java:146)\r\n\tat sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)\r\n\tat sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)\r\n\tat java.lang.reflect.Method.invoke(Unknown Source)\r\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)\r\n\tat org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:564)\r\n\tat org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)\r\n\tat org.testng.internal.Invoker.invokeConfigurations(Invoker.java:138)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:254)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1149)\r\n\tat org.testng.TestNG.run(TestNG.java:1057)\r\n\tat org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)\r\n\tat org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)\r\n\tat org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)\r\n"},"stage":"finished","description":"","start":1521415029693,"stop":1521415051873}],"afters":[{"name":"tearDown","status":"broken","statusDetails":{"known":false,"muted":false,"flaky":false,"trace":"java.lang.NullPointerException\r\n\tat com.rf.core.driver.website.RFWebsiteDriver.quit(RFWebsiteDriver.java:866)\r\n\tat com.rf.test.website.RFLSDWebsiteBaseTest.tearDown(RFLSDWebsiteBaseTest.java:248)\r\n\tat sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)\r\n\tat sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)\r\n\tat java.lang.reflect.Method.invoke(Unknown Source)\r\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)\r\n\tat org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:564)\r\n\tat org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)\r\n\tat org.testng.internal.Invoker.invokeConfigurations(Invoker.java:138)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:322)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:254)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1149)\r\n\tat org.testng.TestNG.run(TestNG.java:1057)\r\n\tat org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)\r\n\tat org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)\r\n\tat org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)\r\n"},"stage":"finished","description":"","start":1521415052067,"stop":1521415052075},{"name":"afterSuite","status":"passed","stage":"finished","description":"","start":1521415052077,"stop":1521415052114}]}